packages/playwright/tsup.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig((options) => ({
+  entry: ['src/index.ts'],
+  splitting: false,
+  minify: !options.watch,
+  format: ['cjs', 'esm'],
+  dts: {
+    resolve: true,
+  },
+  treeshake: true,
+  sourcemap: true,
+  clean: true,
+  platform: 'node',
+  esbuildOptions(options) {
+    options.conditions = ['module'];
+  },
+}));
```

Comment:
Would it be worth extending the shared package's tsup config here (like you did with tsconfig) to avoid things getting out-of-sync that we want to keep throughout the repo (e.g. tree shaking)?

Reply:
Good call! I'm not sure if that's possible but def worth looking into. Mind if that's done in a follow-up?

Reply:
Yes, that's fine!